src/components/TweetsList/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+
+import TweetItem from './TweetItem'
+import Loading from '../Loading';
+import { useTweets } from '../../providers/tweets';
+
+const TweetsList = () => {
+  const { tweets, areLoading, requestTweets } = useTweets();
```

Comment:
Не нужно переменным артикулы добавлять
```suggestion
  const { tweets, loading, requestTweets } = useTweets();
```